src/test/java/de/adesso/kicker/user/UserServiceTest.java

```diff
@@ -51,19 +51,28 @@ void setUp() {
 
     @Test
     void testGetAllUsers() {
+        // when
         ArrayList<User> allUsers = new ArrayList<>(userService.getAllUsers());
+
+        // then
         assertTrue(allUsers.contains(user));
     }
 
     @Test
     void testGetUserById_Success() {
+        // when
         User idUser = userService.getUserById(user.getUserId());
+
+        // then
         assertEquals(idUser, user);
```

Comment:
Same as before. It would be easier to read/understand the test if the `user` would be defined within the method.